src/app/_client-services/next_api_client_service.ts

```
@@ -1081,4 +1089,28 @@ export class NextApiClientService {
+		votesType
+	}: {
+		proposalType: EProposalType;
+		index: string;
+		analyticsType?: EAnalyticsType;
+		votesType: EPostTileVotesType;
+	}) {
+		const queryParams = new URLSearchParams({
+			analyticsType: analyticsType ? analyticsType.toString() : '',
+			votesType: votesType.toString()
+		});
+		const { url, method } = await this.getRouteConfig({ route: EApiRoute.GET_POST_TILES_VOTES, routeSegments: [proposalType, index, 'votes', 'votes-tiles'], queryParams });
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)